lib/create-identity-response-handler.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * The contents of this file are only relevant if your service connects
```

Comment:
How about
'Use this file only if your service connects'
?

Reply:
[as above]